src/RA_Core.cpp

```diff
@@ -1690,3 +1688,23 @@ BOOL CanCausePause()
 {
     return (_RA_CausePause != nullptr);
 }
+
+namespace ra {
+_Use_decl_annotations_
+std::string GameJSONFilename(GameID game_id) noexcept
+{
+    std::ostringstream oss;
+    oss << RA_DIR_DATA << game_id << ".txt";
+    return oss.str();
```

Comment:
With tiny functions like this, `stringstream `doesn't seem that bad but I've been a little hesitant to do it for larger functions.

Reply:
why do you think `stringstream` is bad?

Reply:
This is really strange, it must have been a bug in VC++ because I can't seem to reproduce what I had encountered last time. It's fine I guess.